src/core_plugins/i18n/server/tool/i18n_verify_keys.js

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs';
```

Comment:
@Bargs I have updated the verify translation task to test against registered translations.